mods/aero-tray.wh.cpp

```diff
@@ -16,6 +17,11 @@
 This mod restores the tray overflow from Windows 7 and 8, including the "Customize..." link,
 drawing tray icons with the system theme, and the general appearance.
 
+# IMPORTANT: READ!
+This mod needs to hook into `winlogon.exe` to successfully capture Explorer starting. Please
+navigate to Windhawk's Settings, Advanced settings, More advanced settings, and make sure that
+`winlogon.exe` is in the Process inclusion list.
```

Comment:
You don't need to `@include winlogon.exe` for this, it's enough to add it to the Process inclusion list.

Reply:
For more technical details see the blog post:
https://m417z.com/Implementing-Global-Injection-and-Hooking-in-Windows/
And the high level architecture:
https://github.com/ramensoftware/windhawk?tab=readme-ov-file#technical-details

You want Windhawk to be injected into `winlogon.exe` for this:
![image](https://github.com/ramensoftware/windhawk-mods/assets/4129781/0f831892-15fe-4555-bd7a-46b4231d2852)

But loading your mod which does nothing has no value.